### DISCUSSION TITLE
How to enable prometheus exporter

### DISCUSSION THREAD

**Discussion by IanRudnikov** [inspectIT/inspectit-ocelot#1581] [General]
java -jar inspectit-ocelot-agent-1.16.0.jar 153044 enabledIfConfigured=true 

How to run jar file and prometheus exporter??

**Comment by heiko-holz** [comment]
Hi @IanRudnikov ,

as written in the [documentation](https://inspectit.github.io/inspectit-ocelot/docs/1.16.0/metrics/metric-exporters), you can start the Prometheus exporter by adjusting your command as follows:

```
java -jar inspectit-ocelot-agent-1.16.0.jar 153044 -Dinspectit.exporters.metrics.prometheus.enabled=ENABLED
```

Please consult the [documentation on configuration sources](https://inspectit.github.io/inspectit-ocelot/docs/1.16.0/configuration/configuration-sources) to learn more about the different ways of passing configuration properties to the agent.
